src/app/metadata/features/metadata-browser/metadata-browser.component.html

```diff
@@ -116,9 +133,18 @@
           </button>
         </div>
       </form>
+      <div class="mt-6">
+        <button
+          mat-stroked-button
+          class="toggle-filters w-full"
+          (click)="displayFilters = !displayFilters"
+        >
+          <mat-icon>menu</mat-icon>Close filters
```

Comment:
Since the other two buttons above do not have icons, this one directly below should also not have one. Alternatively, add an icon to all of them. But the "menu" icon looks a bit out of place here, particularly because the same button at the top of the section has different icon(s).

Reply:
Another problem is that the button also appears when this is displayed as sidebar, in which case it does not do anything.